src/app/quiz/components/BooleanInput.js

```diff
@@ -31,15 +31,16 @@ export function BooleanButton({children, onClick, sx}) {
   )
 }
 
-export default function BooleanInput({handleAnswerSubmit, postAnswerIsMutating, ...props}) {
+export default function BooleanInput({question, handleAnswerSubmit, postAnswerIsMutating, ...props}) {
   function handleSubmit(value) {
-    return handleAnswerSubmit({text: value})
+    return handleAnswerSubmit({choices: [value]})
```

Comment:
But booleaninput is not actually a choice input. I supposed to use a boolean field to retain a value here. 
I would leave untoched it in this PR.